File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    commands.map((a) => {
        answer.push(array.slice(a[0] - 1, a[1]).sort(((a, b) => a - b))[a[2] - 1]);
        
    })
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        ArrayList<Integer> list = new ArrayList<>();
        
        for(int[] command : commands) {
            int[] newArr = Arrays.copyOfRange(array, command[0] - 1, command[1]);
            Arrays.sort(newArr);
            list.add(newArr[command[2] - 1]);
        }
        
        return list.stream().mapToInt(Integer::intValue).toArray();
    }
}